### PR TITLE
Home page 

### PR DESCRIPTION
**Pull request by Therealadetunji**
Home-page done with html, css and javacript
javascript used for the mobile menu
The link to my loom video below

https://www.loom.com/share/db777c58120b4eeb86e2d0c700551c85

### CODE REVIEW

**Comment by Therealadetunji** [comment]
https://www.loom.com/share/db777c58120b4eeb86e2d0c700551c85

File: About-page/index.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            
+              </div>
+
+             <div class="photograph-desktop">
+                <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="desktop-image">
+             </div>
+
+            </div>
+          </div>
+    </section>
```
Comment (vic778) [inline About-page/index.html:152]: - [ ] Kindly for each div of this section must have an icon, title and the paragraph as you can see at the screenshot 
 Your template  
Screenshot from 2022-07-02 00-54-23

Figma template 

Screenshot from 2022-07-02 00-55-08

File: About-page/index.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        <div class="partners">
+            <div><h2 class="sponsor1">Google</h2></div>
+            <div><h2 class="sponsor2">FIFA</h2></div>
+               <div><h2 class="sponsor3">UEFA</h2></div>
+        </div>
+        <div class="english-sponsor"> 
+            <h2>English Premier League</h2>
+        </div>
+
+    </section>
```
Comment (vic778) [inline About-page/index.html:172]: - [ ] Kindly you could have a div that contain all information for a particular speaker then a div that contain the image of the speaker and the background image then i second div that contain the name the `p` that must have the color `rgb(236, 82, 66)`  and the last `p` <br>
like `div class="container"> <div class="image">...</div> <div class="details">...</p></div> </div>`
Your Design
Screenshot from 2022-07-02 01-13-29

Figma design

Screenshot from 2022-07-02 01-13-44

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @Therealadetunji 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: About-page/index.html (hunk 1 above)
Comment (roshan-bajgain) [inline About-page/index.html:152]: - [ ] [Optional] You have well implemented the required changes said by the previous reviewer but if you look at the `Behance` design, there is equal padding from all sides. Kindly implement it
### your design
his
### Behance design

figam

File: About-page/index.html (hunk 2 above)
Comment (roshan-bajgain) [inline About-page/index.html:172]: - [ ] Kindly implement the requested by the previous reviewer that should have an image, title, and content. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description in 3 rows.
### Your design
youuuu
### Orignal design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Therealadetunji, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
Hi @Therealadetunji, 

Well done making the changes requested by the previous reviewer 🚀
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it 🔥 

### Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**